.zenodo.json

```diff
@@ -563,6 +563,11 @@
       "affiliation": "MIT, HMS",
       "name": "Ghosh, Satrajit",
       "orcid": "0000-0002-5312-6729"
+    },
+    {
+      "affiliation": "University College London",
+      "name": "Mancini, Matteo",
+      "orcid": "0000-0001-7194-4568"
```

Comment:
I'm not sure what the general ordering policy is, but I'm pretty sure Satra should be the last author

Reply:
Apologies, I thought the file was in some chronological order of contributions.

Reply:
Looks like @mgxd (or someone) reorders it later anyway, so fixing it won't save anybody time. https://github.com/nipy/nipype/pull/2338#issuecomment-356027541. Sorry for the noise.